### ISSUE TITLE
Document release process

### ISSUE THREAD

**Issue by letmaik** [TimothyClaeys/pycose#99]
I think this is roughly:
1. Ensure `version` is set to the next SemVer version in setup.cfg
2. Create a new GitHub release with git tag `v<x>.<y>.<z>`, e.g., `v1.2.3`
3. Wait for CI to run for the new git tag
5. Make sure that CI ran successfully and pushed the package to PyPI

**Comment by TimothyClaeys** [comment]
One other thing I can think off is to make sure that the documentation is up to date and verify if it is successfully deployed to [pycose.readthedocs.io](https://pycose.readthedocs.io/en/latest/index.html).

**Comment by letmaik** [comment]
Good point, it says "latest" on the docs, shouldn't it include the version? Otherwise it becomes a little hard to check.
